server/src/api/schemas.py

```diff
@@ -31,6 +31,15 @@ class AgentIn(Schema):
     log = fields.List(fields.String(), required=False)
 
 
+class AgentOut(Schema):
+    """Agent data input schema"""
+
+    state = fields.String(required=False)
+    queues = fields.List(fields.String(), required=False)
+    location = fields.String(required=False)
+    job_id = fields.String(required=False)
```

Comment:
does `required=False` mean that the returned json from this endpoint might not include this field at all?

Reply:
it's possible that any of those fields might not be populated.  I'm not sure off the top of my head whether the GET would fail if it's not present, but at the very least, it communicates this possibility in the swagger docs.